data/categories.txt

```diff
@@ -10203,6 +10203,44 @@ sk:Mobilný operátor
 fa:اپراتور تلفن همراه
 mr:दूरसंचार कंपनी
 
+craft
+en:Craft
+ar:حرفة
+be:Рамяство
+bg:Занаяти
+cs:Řemeslo
+da:Håndværk
+de:Handwerk
+el:Τεχνική
+es:Gremios
+et:Käsitöö
+eu:Artisautza
+fa:صنعت‌گری
+fi:Käsityö
+fr:Artisanat
+he:מְלָאכָה
+hu:Kézművesség
+id:Kerajinan
+it:Artigianato
+ja:工房
+ko:공예
+mr:हस्तकला
+nb:Håndverk
+nl:Ambacht
+pl:Rzemiosło
+pt:Ofícios
+ro:Meșteșuguri
+ru:Мастерская
+sk:Remeselník
+sv:Hantverk
+sw:Ufundi
+th:งานฝีมือ
+tr:Zanaat
+uk:Майстерня
+vi:Thủ công
+zh-Hans:工艺作坊
+zh-Hant:工藝作坊
```

Comment:
Well, should either add ```@craft``` category into every craft-* entry or remove this generic term.

I prefer to remove it now.

Reply:
Removed from `categories.txt`. 

`[type.craft]` was already in `types_strings.txt`.